content-services/latest/install/ansible.md

```diff
@@ -0,0 +1,489 @@
+```
+
+Alternatively, to install an earlier version of ACS Enterprise (e.g. 6.2.2)  system use the following command:
+
+```bash
+ansible-playbook playbooks/acs.yml -i inventory_local.yml -e "@6.2.N-extra-vars.yml"
+```
+
+> NOTE: The playbook takes around 30 minutes to complete.
+
+Once the playbook is complete Ansible will display a play recap to let you know that everything is done, similar to the block below:
+
+```bash
+PLAY RECAP *******************************************************************************************************
+activemq_1                 : ok=24   changed=0    unreachable=0    failed=0    skipped=17   rescued=0    ignored=0
+adw_1                      : ok=24   changed=6    unreachable=0    failed=0    skipped=6    rescued=0    ignored=0
+database_1                 : ok=20   changed=0    unreachable=0    failed=0    skipped=11   rescued=0    ignored=0
+nginx_1                    : ok=21   changed=8    unreachable=0    failed=0    skipped=8    rescued=0    ignored=0
+repository_1               : ok=92   changed=43   unreachable=0    failed=0    skipped=14   rescued=0    ignored=0
+search_1                   : ok=34   changed=13   unreachable=0    failed=0    skipped=11   rescued=0    ignored=0
+syncservice_1              : ok=39   changed=18   unreachable=0    failed=0    skipped=13   rescued=0    ignored=0
+transformers_1             : ok=81   changed=10   unreachable=0    failed=0    skipped=44   rescued=0    ignored=0
+```
+
+Once ACS has initialized access the system using the following URLs with a browser:
+
+* Digital Workspace: `http://<control-node-public-ip>/workspace`
+* Share: `http://<control-node-public-ip>/share`
+* Repository: `http://<control-node-public-ip>/alfresco`
+* API Explorer: `http://<control-node-public-ip>/api-explorer`
+
+# Remote Installation
+
+To install to hosts other than the control node an SSH connection is required. The control node must have network access to all the target hosts and permission to SSH into the machine.
+
+> **Note:** The inventory file (`inventory_ssh.yml`) is used to specify the target IP addresses and the SSH connection details. You can specify one IP address for all the hosts to obtain a single-machine installation, or different IP addresses for a multi-machine installation.
+
+The example snippet below demonstrates how to install the repository to a host with an IP address of `50.6.51.7` and SSH key at `/path/to/ssh_key.pem`.
+
+```yaml
+repository:
+  hosts:
+    repository_1:
+    connection: shh
+    ansible_host: 50.6.51.7
+    ansible_private_key_file: "/path/to/ssh_key.pem"
+    ansible_user: centos
+    ansible_ssh_common_args: "-o UserKnownHostsFile=/dev/null -o ControlMaster=auto
+      -o ControlPersist=60s -o ForwardX11=no -o LogLevel=ERROR
+      -o IdentitiesOnly=yes -o StrictHostKeyChecking=no"
+```
+
+
+## Single Machine Installation
+
+The diagram below shows the result of a single machine installation.
+
+![](../../images/acs-single-machine.png)
```

Comment:
Image is not visible and I think the image link is not specified correctly, should be something like:
![acs-single-machine]({% link content-services/images/acs-single-machine.png %})